### DISCUSSION TITLE
What's the difference between nci.devShells.default and the project/workspace devShell?

### DISCUSSION THREAD

**Discussion by NickHu** [90-008/nix-cargo-integration#118] [Q&A] [answered]
If you comment/uncomment this line:
https://github.com/yusdacra/nix-cargo-integration/blob/954f248c142e3f174057cdc11f1c1cd6903c3048/examples/simple-workspace/flake.nix#L45
you get different devShells (at least, the name of the devShell is different). I can understand that the project devShell is supposed to be the union of all the crate devShells contained within that project, but if you have that line commented out you get some kind of "default" devShell. Is the intention that nix-cargo-integration actually allows you to specify more than one project/workspace in a flake? If so, this is not clear from the `simple-workspace` example, as there doesn't seem to be any information in the `flake.nix` which ties `my-crate` and `my-other-crate` to `my-project`.

**Comment by 90-008** [comment, marked as answer]
That was a bug, the "default" devshell appearing that is. I have fixed that in latest master. And yes, NCI does allow specifying multiple projects. A project can be a crate or a workspace. I can add an example showcasing that if it's not clear.
